assets/app/views/_template.html

```diff
@@ -0,0 +1,7 @@
+<li class="template">
+  <img ng-src="{{template.icon}}">
```

Comment:
is template.icon always going to be defined?  is that something thats going to actually be stored on the api object or just something you'll be setting somewhere in the controller?